src/amo/components/Errors/NotFound/index.js

```diff
@@ -56,6 +58,20 @@ export class NotFoundBase extends React.Component<InternalProps> {
             </Link>
           ),
         ],
+        [
+          'communityStart',
+          'communityEnd',
+          (text) => (
+            <Link
```

Comment:
Do you think this should open in a new tab, as it's an external site?

Reply:
Good question, do we always open external links in new tabs? As a user, I don't like websites that do this. <kbd>ctrl</kbd> + click FTW :)

Reply:
I don't think we're consistent, but I know that for the new hero elements, for example, we do open external links in a new tab. I suppose if we assume that a common use case for the 404 page is that a user has arrived from an external site, via a link, and they are going to proceed to discourse, then it makes sense to *not* open the link in a new tab.